lib/wlib/comm/PacketConstructor.cpp

```diff
@@ -0,0 +1,114 @@
+/** @file PacketConstructor.cpp
+ *  @brief Creates a packet for sending data from the slave systems to the main computer.
+ *
+ *  Implements the functions defined in PacketConstructor.h.
+ *
+ *  @author Heather D'Souza (heathkd)
+ *  @date November 7, 2017
+ *  @bug No known bugs.
+ */
+
+#include <stdlib.h>
+//#include "../stl/Bitset.h"
+#include "PacketConstructor.h"
+#include <iostream>
+
+PacketConstructor::PacketConstructor(const float* p_data, const Bitset<2> &packetType, const Bitset<6> &packetName)
+        : m_pdata{p_data}, m_packetType{packetType}, m_packetName{packetName}  {
+    m_pdata = p_data;
+}
+/*
+const float* PacketConstructor::getData() const {
```

Comment:
For things that are commented out, unless they have a use uncommented, please just remove them.